shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IConfiguration.ts

```diff
@@ -80,4 +80,14 @@ export interface IConfiguration {
      * If channels are provided here, core will ignore any channels that are already setup, example if there is a SKU with an initialized channel
      */
     channels?: IChannelControls[][];
+
+    /**
+     * [Optional] Send performance timing events to the notification manager
+     */
+    enablePerfMgr?: boolean;
```

Comment:
Should this be clarified that it doesn't do anything with telemetry or the `performance` object? This seems to be for debugging only